### DISCUSSION TITLE
Black/White List on architecture tests

### DISCUSSION THREAD

**Discussion by DhavalGondaliya9** [pestphp/pest#1034] [Ideas]
I have figured out while implementing the model extend architecture test.

We are using our custom model and I want to add a test that no one will extend the Laravel Model directly. But I want to exclude some of the files.

I checked the whole documentation and code. I haven't found it anyway.

For Example : 

test('models should extend base model')
    ->expect('App\Models')
	->exclude(['App\Models\User', 'App\Models\Post'])
    ->toExtend('App\Models\Model');

**Comment by owenvoke** [comment]
It looks like you probably want to use [the `ignoring()` method](https://pestphp.com/docs/arch-testing#content-ignoring)? 🤔 This allows you to ignore a specific class or namespace.

```php
test('models should extend base model')
  ->expect('App\Models')
  ->toExtend('App\Models\Model')
  ->ignoring(['App\Models\User', 'App\Models\Post']);
```

**Comment by DhavalGondaliya9** [comment]
Thank You...
